CorsixTH/Lua/entities/machine.lua

```diff
@@ -22,8 +22,6 @@ local TH = require("TH")
 
 corsixth.require("announcer")
```

Comment:
As per other comment.
You don't need this line anymore if you're taking any announcements out of machine.